source/JABHandler.py

```diff
@@ -746,6 +774,8 @@ def initialize():
 	except WindowsError:
 		raise NotImplementedError("dll not available")
 	_fixBridgeFuncs()
+	if config.isInstalledCopy() and not isBridgeEnabled():

```

Comment:
We might have to avoid doing this at secure screens:

```suggestion
	if not globalVars.appArgs.secure and config.isInstalledCopy() and not isBridgeEnabled():
```

Reply:
Nice catch! Thanks.